docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,65 +1,124 @@
 = Using the Data Migration Tool
-:description: This guide explains how to migrate data from Hazelcast 4.x or 5.x clusters to newer 5.x clusters.
+:description: You can use the Data Migration Tool (DMT) to migrate your data from version 4.x or 5.x Open Source and Enterprise Edition Hazelcast clusters when upgrading to 5.3.x versions of Enterprise Edition, or moving to the latest {hazelcast-cloud} release.
 
-[blue]*Hazelcast Enterprise Feature*
+{description} 
 
```

Comment:
Can we add a note on typical use cases, something like:

The Data Migration Tool is typically used for:
- migration from open-source Hazelcast cluster to Hazelcast Enterprise cluster
- migration from an older version of Hazelcast to a newer version of Hazelcast, possibly skipping several minor versions
- migration from an on-premise cluster to a self-managed Hazelcast Enterprise cluster in the cloud or in Viridian (managed Hazelcast)
- selective migration of application data from one cluster to another due to infrastructure changes (infrastructure consolidation or separation)

And maybe mention that for 
- no-downtime, in-place upgrade the rolling upgrade feature should be used
- 3.12-> 5.x migration the migration jar should be used

Reply:
Aren't the typical use cases (with the possible exception of selective migration, which can be added) covered in the first para explaining what you can use the DMT for?

We have said that we recommend that the source cluster is put into PASSIVE mode, doesn't this exclude the idea of no-downtime? Without some downtime, we can't guarantee everything will be migrated, so we ought not tell them how to to it.

We state that the tool only supports from version 4.x, there are separate topics for migrating/upgrading from 3.12 - I can add a note that if this is what they are trying to do they should see those topics.

Reply:
Added selective migration to first para

Added note about upgrading/migrating 3.12.x

Reply:
I am not sure if the current first paragraph makes all the options clear, e.g. migration EE 5.3.5 -> EE 5.3.5 is supported, but is neither an upgrade nor a move to Viridian.

For no downtime a different feature shuld be used, see below.

>note that if this is what they are trying to do they should see those topics.

this is what I meant by the last two bullet points
- no-downtime, in-place upgrade -> https://docs.hazelcast.com/hazelcast/5.3/maintain-cluster/rolling-upgrades#hide-nav
- 3.12 -> 5.x -> https://docs.hazelcast.com/hazelcast/5.3/migrate/migration-tool-imdg

Reply:
OK, added the list of typical uses & notes to cover the last bullet points